src/h5web/toolbar/controls/DomainSlider/DomainSlider.module.css

```diff
@@ -22,6 +25,12 @@
   pointer-events: none;
 }
 
+.trackWrapper {
+  flex: 1 1 0%;
+  display: flex;
+  padding: 0 calc(var(--thumb-size) / 2); /* make track end in middle of thumbs */
```

Comment:
Can't you simply add a `margin: 0 calc(var(--thumb-size) / 2);` to `.track` so that you don't need the wrapper ?

Reply:
Not at all a dumb question. The wrapper was left over from a previous attempt. I thought a `padding` was better at conveying what we were trying to do, but it's not worth the extra wrapper you're right.